src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -170,22 +131,13 @@ export const BaseComment = ({
         commentFormSubmitHandler();
     };
 
-    const handleStatusUpdate = (selectedStatus: FeedItemStatus): void => {
-        if (onAnnotationEdit) {
-            onAnnotationEdit({ id, permissions });
-        } else if (onCommentEdit) {
-            onCommentEdit({ id, status: selectedStatus, hasMention: false, permissions });
-        }
-    };
-
     // Since we have to pass onCommentEdit through annotations (to Replies), onAnnotationEdit essentially overrides onCommentEdit
     const onEdit = onAnnotationEdit ?? onCommentEdit;
 
-    const canDelete = permissions.can_delete;
+    const canDelete = !!permissions.can_delete;
```

Comment:
Does the API return non-bool values? I've only seen bool permissions

Reply:
The type on the permissions object is `boolean | undefined`.
```
type BoxCommentPermission = {
    can_delete?: boolean,
    can_edit?: boolean,
    can_reply?: boolean,
    can_resolve?: boolean,
};
```
Turns out that's true for the other permissions too, so I've done that as well (below). I'm assuming it can be undefined because the object can get instantiated without or before a 200.